runners/client_wrapper_test.go

```diff
@@ -0,0 +1,78 @@
+package runners
+
+import (
+	"context"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+	"github.com/topolvm/pvc-autoresizer/client"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("test clientWrapper", func() {
+	It("test metrics", func() {
+		m := *mgr
+		wrappedClient := client.NewClientWrapper(m.GetClient())
```

Comment:
I prefer using a mock client that always fails for NewClientWrapper.
Do you need to use real client?